README.md

```diff
@@ -0,0 +1,116 @@
+  <strong>
+    <h1 align="center" >create-express-ts-api</h3>
+<p align="center">
+    <strong>
+      A starter to kit for developing  REST API in typescript and express.
+  </p>
+<!-- PROJECT SHIELDS -->
+<br></br>
+
+![restapi](https://user-images.githubusercontent.com/72249059/197589566-b2de5f90-1ddb-4d89-9811-e4da022bdd54.jpg)
+
+
+<br />
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#getting-started">Getting Started </a>
+    </li>
+    <li>
+      <a href="#Tech-Stack-Used">Tech stack used</a>
+    </li>
+    <li><a href="#Understanding-the-folder-structure">Understanding the folder structure</a></li>
+    <li><a href="#Setting-up-Development-DB">Setting up Development DB</a></li>
+    <li><a href="#Documentation">Documentation</a></li>
+  </ol>
+</details>
+
+---
+## Getting Started
+
+Follow the instructions to set up the project on your local machine.
+
+### Prerequisites
+
+Install [NodeJS LTS](https://nodejs.org/en/)
+
+- npm
+
+  ```sh
+  npm install npm@latest -g
+  ```
+
+### Initializing the project
+
+1. Initiate your project by using the following npm command:
+
+   ```sh
+   npx create-express-ts-api  <project-name>
+   ```
+
+   example : `npx create-express-ts-api Firstapi`
+
+3. Install NPM packages
+
+   ```sh
+   npm install
+   ```
+
+4. Start the react and nodejs server concucrrently
+
+   ```sh
+   npm run dev
+   ```
```

Comment:
At this step provide directions to add a .env file and add
``PORT=3000``

In the next step, ask to set up the development DB and add a hyperlink to set up the development DB section of the same readme file

Also, we don't have any react server in this project, so pls change this statement to ``Start the server by running the following command`` and this will be our last step. 

Add a note at the end of this section, if facing any issues or feel free to raise a discussion or issue or head towards contact section (add hyperlink to contact section only)

Reply:
done